tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "sourceMap": true,
+    "removeComments": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": false,
+    "lib": [
+      "es7",
+      "dom"
+    ],
+    "jsx": "react"
+  },
+  "include": [
+    "front-end/src/**/*.tsx"
```

```suggestion
    "front-end/src/**/*"
```

Let's remove the extension. There might be other files as well.